src/frontend/containers/Settings/index.tsx

```diff
@@ -424,16 +424,34 @@ const BackgroundProcesses = observer(() => {
   );
 });
 
-const Shortcuts = () => (
-  <>
-    <h2>Keyboard shortcuts</h2>
-    <p>
-      Click on a key combination to modify it. After typing your new combination, press Enter to
-      confirm or Escape to cancel.
-    </p>
-    <HotkeyMapper />
-  </>
-);
+const Shortcuts = observer(() => {
+  const { uiStore } = useContext(StoreContext);
+  return (
+    <>
+      <h2>Click behavior</h2>
```

Comment:
Feels weird here. Personally, I am against this feature. There is a big fat button saying return. It would make sense if the slide mode was some kind of popup/modal but right now we treat it as just another view like masonry or gallery view. Thinking about it, a popup/modal wouldn't be such a bad idea and the preview windowcould become the slide mode. Basically, you could separate it like those docker panels a la Gimp.

Reply:
I think it does feel more intuitive to dbl click to go both in and out, maybe we should get some more opinions on this.
In this PR I've made it work like an overlay internally already so that could be a good solution, but it might not mesh well with the inspector